lib/utils.js

```diff
@@ -225,3 +225,12 @@ function crawl(object, visitor) {
     crawl(object[key], visitor);
   }
 }
+
+exports.ensureForwardSlashesInPath = (path) => {
```

Comment:
I would use this implementation https://github.com/sindresorhus/slash/blob/main/index.js

Reply:
Wasn't aware of that package. I used the implementation from [openapi-cli `utils.ts`](https://github.com/Redocly/openapi-cli/blob/b389661f91202174d5c46766ef2be73bfe9faf0d/packages/core/src/utils.ts#L163) which is the same except it does not check for non-ascii symbols. 

What would be the benefit to also check for unicode in this use case?

Reply:
Oh, I think we copied it into openapi-cli for a reason. Then keep it as is :D, just add a comment it was copied from openapi-cli.

Reply:
Done :) 
https://github.com/Redocly/create-openapi-repo/pull/82/commits/37670dc3cc2efe5a499fc20d868cc14b2e3aa4e2